Civi/ActionSchedule/RecipientBuilder.php

```diff
@@ -283,7 +283,7 @@ protected function prepareQuery($phase) {
     /** @var \CRM_Utils_SQL_Select $query */
     $query = $this->mapping->createQuery($this->actionSchedule, $phase, $defaultParams);
 
-    if ($this->actionSchedule->limit_to /*1*/) {
+    if ($this->actionSchedule->limit_to == 2) {
```

Comment:
For consistency with the other comparison: if ($this->actionSchedule->limit_to === 2)